docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,102 @@
+# Release 434 (20 Nov 2023)
+
+## General
+
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related session property `max_writer_tasks_count` to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+* Support reading `json` type. ({issue}`19790`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance when reading large checkpoint files on partitioned tables.
+  ({issue}`19588`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Add timestamp to date schema evolution. ({issue}`19513`)
```

Comment:
I think I've been documenting this as support for reading columns that have been changed from x type to y type, so we might want to change past notes or this note so they're consistent. Also, `timestamp` and `date`.

Reply:
Done